src/Microsoft.Azure.Functions.ExtensionBundle/bundleConfig.json

```diff
@@ -1,6 +1,6 @@
 ﻿{
     "bundleId":  "Microsoft.Azure.Functions.ExtensionBundle",
-    "bundleVersion":  "4.13.1",
+    "bundleVersion":  "4.15.0",

```

Comment:
I see 4.14.0 was already in the release notes, so I went ahead and increased all the way to 4.15.0

Reply:
lets do 4.13.2 instead

Reply:
done!